SocialCareCaseViewerApi/V1/UseCase/FormSubmissionsUseCase.cs

```diff
@@ -121,14 +121,16 @@ private static void UpdateSubmissionState(CaseSubmission submission, UpdateCaseS
                 { "in_progress", SubmissionState.InProgress },
                 { "submitted", SubmissionState.Submitted },
                 { "approved", SubmissionState.Approved },
-                { "discarded", SubmissionState.Discarded }
+                { "discarded", SubmissionState.Discarded },
+                { "panel-approved", SubmissionState.PanelApproved }
```

Comment:
let's use an underscore to be consistent?

Reply:
sure